src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -7,6 +11,21 @@
     'CEA-608'
 ]
 
+def save_json_to_file(caption_data: dict):
+    # datetime object containing current date and time
```

Comment:
add a docstring for the function stating what it does, look at the below function for reference

Reply:
Done